src/main/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoService.java

```diff
@@ -170,4 +178,16 @@ private int getCcdClaimsCount(final SearchResult ccdClaims) {
 
         return Optional.ofNullable(ccdClaims).map(SearchResult::getTotal).orElse(0);
     }
+
+    private DashboardClaimStatus getStatus(boolean isClaimant, CaseData caseData) {
+        return isClaimant
+            ? dashboardClaimantClaimStatusFactory.getDashboardClaimStatus(new CcdDashboardClaimantClaimMatcher(
+            caseData,
+            featureToggleService
+        ))
+            : dashboardClaimStatusFactory.getDashboardClaimStatus(new CcdDashboardClaimMatcher(
```

Comment:
Maybe the existing claim matcher could be renamed to CcdDashboardDefendantClaimMatcher if it is only to be used for defendant.

Reply:
done